src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14SwitchExpression.java

```diff
@@ -1,12 +1,45 @@
 //non-compiled with javac: Compilable with Java14
-package com.puppycrawl.tools.checkstyle.grammar;
+package com.puppycrawl.tools.checkstyle.grammar.java14;
 
+import static java.time.Instant.*;
+
+import java.time.Instant;
+import java.util.Arrays;
 import java.util.EnumSet;
 import java.util.Set;
+import java.util.function.IntConsumer;
 import java.util.function.Supplier;
 
 public class InputJava14SwitchExpression {
 
+    static class HardToParse {
+        void foo() {
+            Instant Ascii;
+            byte[] decodabet = new byte[1];
+            Arrays.fill(decodabet, (byte) -1);
+            char[] chars = {'c','h','a','r','s'};
+            for (
+                    int i = 0;
+                    i < chars.length; i++) {
+                char c = chars[i];
+                decodabet[c] = (byte) i;
+            }
+        }
+        IntConsumer consumer = (IntConsumer) i -> {
+        };
+
+        static String yield = "yield";
+
+        class yieldClass{
+            int yield = 6;
+
+            public void yield(){
+
+            }
+        }
+
+    }
+
```

Comment:
@rnveach

Reply:
Based on the grammar changes, you would need the `yield` usage in the `switch` and not just in random places, imo.

Reply:
That will not compile in jdk14.

Reply:
https://github.com/checkstyle/checkstyle/pull/8449/files#r468609715

Reply:
And here: https://github.com/checkstyle/checkstyle/pull/8449/files#r468615917